.github/workflows/test-aerospike.yml

```diff
@@ -0,0 +1,38 @@
+on:
+    push:
+        branches:
+            - master
+            - main
+        paths:
+            - 'aerospike/**'
+    pull_request:
+        paths:
+            - 'aerospike/**'
+name: "Tests Aerospike"
+jobs:
+    Tests:
+        runs-on: ubuntu-latest
+        services:
+            aerospike:
+                image: 'aerospike/aerospike-server:latest'                
+                ports:
+                    - '3000:3000'
+                    - '3001:3001'
+                    - '3002:3002'
+                    - '3003:3003'
```

Comment:
bug: this service is not needed anymore, as the container will be started by testcontainers-go
```suggestion
```